entities/03_for_statement.coffee

```diff
@@ -0,0 +1,13 @@
+class ForStatement
```

Comment:
Yes - that way the various entities are ordered by their level or precedence. I thought it would make organization easier.